# The position of all the points of interest 
poi_x: [183.46, 161.74, 129.96, 106.88, 151.45, 102.53, 86.76, 58.87, 11.33, 125.16, 96.82, 73.73, 80.59, 56.13, 35.33, -16.56, -39.65, 61.84, 49.04, 27.33, 4.7, -72.57, -94.28, 41.96, 19.1, -4.22, -25.71, -47.65, -123.54, -145.49, 10.64, -5.13, -41.25, -33.48, -55.65, -77.14, -100.46, -194.18, -64.57, -115.31, -86.51, -140.46, -107.09, -128.8, -150.52, -172.46, -223.67, -243.1, -160.35, -198.29, -180.23, -201.95, -295.67, -271.44, -218.87, -237.61, -292.02, -257.27, -312.59, -277.84, -799.38, -348.2, -505.1, -315.22, -256.32, -243.82, -164.53, -146.75, -113.16, -68.03, -395.89, -264.41, -195.81, -119.84, -147.38, -116.96, -118.36, -45.27, -64.06, -99.1, -557.59, -320.2, -264.46, -284.85, -250.2, -208.75, -228.8, -176.0, -134.2, -189.01, -155.08, -153.62, -82.25, -101.78, -125.33, -270.01, -380.77, -50.59, -7.34, -59.83, -27.74, -37.29, -58.22, -80.92, -133.75, -112.23, -89.77, -4.32, 14.55, -38.39, -14.0, 22.82, 49.87, -30.77, 46.63, 25.01, 1.7, 125.65, 189.08, 81.89, 134.51, 104.23, 788.52, 164.83, 383.95, 106.19, 193.44, 157.25, 264.41, 114.33, 193.3, 217.79, 1199.38, 56.27, -21.1, -13.03, 29.28, 6.86, 39.31, 81.74, 61.75, -26.01, 10.12, 53.87, -196.23, -170.93, -148.25, -119.51, -43.05, -76.49, -147.22, -126.48, -101.91, -52.84, -74.21, -94.14, -246.65, -219.29, -269.88, -177.46, -228.82, -198.65, -209.05, -244.61, -170.48, -189.76, -267.05, -223.72, 47.17, 67.49, 74.71, 138.8, 678.22, 97.92, 36.44, 12.77, 100.41, 58.69, 164.42, 20.85, -21.07, 56.14, 73.71, 32.37, 0.72, 172.6, 156.71, 134.31, 140.71, 126.99, 96.82, 72.82, 35.1, 116.17, 108.02, 99.45, 91.56, 68.36, 45.73, 9.38, -14.16, 84.49, 75.51, 61.39, 38.19, 16.01, -33.94, -44.57, -66.97, 51.9, 30.53, 7.44, -14.96, -36.68, -85.6, -109.03, -119.89, 2.75, -15.31, -19.31, -44.57, -66.4, -88.8, -112.0, -134.52, -169.83, -52.91, -89.94, -96.8, -113.49, -117.94, -139.66, -161.49, -183.32, -214.05, -209.41, -203.31, -218.64, -127.89, -170.29, -185.44, -178.42, -173.43, -191.09, -212.81, -269.39, -234.87, -228.24, -245.16, -264.81, -267.56, -284.93, -286.76, -302.3]
poi_y: [-354.84, -535.28, -288.03, -463.36, -129.72, -64.19, -625.92, -781.25, -720.82, 111.57, 355.42, 183.91, -223.35, -398.68, -556.13, -486.77, -658.27, 571.17, 2.2, -156.96, -328.46, -423.36, -588.48, 424.78, 248.6, 74.97, -87.59, -259.09, -357.4, -524.64, 657.56, 480.96, 733.74, 309.88, 141.78, -20.78, -195.68, -462.94, 562.24, 624.79, 376.69, 446.06, 208.17, 47.31, -125.89, -290.58, -228.03, -399.95, 281.37, 519.26, 112.85, -56.1, -313.56, 420.1, 350.31, 181.79, 248.17, 13.27, 83.06, -155.26, 221.06, -62.87, -723.28, -223.3, 998.85, 809.14, 882.48, 544.62, 528.24, 655.11, -623.58, -554.66, -564.83, -766.82, -633.0, -979.85, -998.85, -564.51, -514.74, -498.77, 275.85, -48.72, 104.41, 158.08, -78.34, 36.24, 89.99, 21.65, 135.35, 204.76, 190.89, -33.23, 67.76, 120.18, -45.52, 559.52, 375.21, 53.26, 168.66, 232.81, 218.14, 405.75, 462.37, 285.72, 356.75, 298.96, 475.3, 386.81, 343.88, 641.15, 574.44, 785.65, 797.01, 0.93, 100.01, 153.61, -13.94, -201.79, -231.63, -317.55, -383.04, -369.22, 236.43, 0.94, -135.62, -150.76, -453.75, -438.66, -845.83, -552.15, -778.56, -814.06, 523.88, -693.3, -904.28, -579.55, -469.87, -630.16, -646.55, -538.18, -484.74, -398.27, -415.08, -304.01, -371.12, -382.17, -434.77, -447.84, -350.19, -332.61, -214.83, -269.38, -102.17, -166.17, -114.97, -285.38, -308.79, -319.51, -241.0, -200.86, -135.05, -148.26, 258.65, 272.36, 372.11, 426.84, 328.05, 441.17, 329.21, 277.96, 86.79, 244.69, 462.01, 30.8, 505.26, 566.16, 475.14, 660.47, 511.1, -69.18, -180.26, -85.87, -135.23, -250.19, -235.72, -445.06, -321.44, -499.32, -208.88, -96.95, -544.64, -703.59, -751.04, 188.95, 259.05, 147.74, -143.77, -311.01, -477.4, -521.45, -689.55, 431.49, 486.86, 93.06, -77.38, -242.71, -375.91, -455.06, -623.37, 497.98, 336.69, 161.79, -6.31, -173.34, -308.24, -474.0, -556.56, 569.26, 695.65, 395.42, 225.83, 60.5, -108.23, -276.54, -441.02, -493.79, 647.99, 593.52, 292.43, 411.38, 127.74, -39.29, -208.24, -376.76, -304.65, -341.63, -390.19, -431.44, 535.43, 197.11, 438.67, 394.65, 363.41, 28.37, -142.06, -356.76, 469.68, 266.05, 385.2, 214.98, -71.0, 48.16, -234.41, 165.62]
poi_z: [-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000,-0.500000000000000]